src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -57,15 +58,21 @@
     protected $camelizedAttributes = array();
 
     /**
+     * @var PropertyInfoExtractor
+     */
+    protected $propertyInfoExtractor;
+
+    /**
      * Sets the {@link ClassMetadataFactoryInterface} to use.
      *
      * @param ClassMetadataFactoryInterface|null $classMetadataFactory
      * @param NameConverterInterface|null        $nameConverter
      */
-    public function __construct(ClassMetadataFactoryInterface $classMetadataFactory = null, NameConverterInterface $nameConverter = null)
+    public function __construct(ClassMetadataFactoryInterface $classMetadataFactory = null, NameConverterInterface $nameConverter = null, PropertyInfoExtractor $propertyInfoExtractor = null)
```

Comment:
Must type hint the `PropertyInfoExtractorInterface` instead of the implementation.